Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision b7f554eebc040791c00214c383a21d6a0ae839ca)
+++ /dev/null	(revision b7f554eebc040791c00214c383a21d6a0ae839ca)
@@ -1,3 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
Index: app/src/main/res/values/strings.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><resources>\n    <string name=\"app_name\">Lab2</string>\n    <string name=\"my_name\">Марія Вергун КП-92</string>\n    <string name=\"btn_1\">1</string>\n    <string name=\"btn_2\">2</string>\n    <string name=\"btn_3\">3</string>\n    <string name=\"btn_4\">4</string>\n    <string name=\"btn_5\">5</string>\n    <string name=\"btn_6\">6</string>\n    <string name=\"btn_7\">7</string>\n    <string name=\"btn_8\">8</string>\n    <string name=\"btn_9\">9</string>\n    <string name=\"btn_0\">0</string>\n    <string name=\"plus\">+</string>\n    <string name=\"minus\">-</string>\n    <string name=\"divide\">/</string>\n    <string name=\"multiply\">*</string>\n    <string name=\"equals\">=</string>\n    <string name=\"clear\">C</string>\n</resources>
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/app/src/main/res/values/strings.xml b/app/src/main/res/values/strings.xml
--- a/app/src/main/res/values/strings.xml	(revision b7f554eebc040791c00214c383a21d6a0ae839ca)
+++ b/app/src/main/res/values/strings.xml	(date 1631286247000)
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Lab2</string>
+    <string name="app_name">Lab1</string>
     <string name="my_name">Марія Вергун КП-92</string>
     <string name="btn_1">1</string>
     <string name="btn_2">2</string>
Index: .gradle/buildOutputCleanup/cache.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>#Sat Sep 25 20:14:43 EEST 2021\ngradle.version=7.0.2\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gradle/buildOutputCleanup/cache.properties b/.gradle/buildOutputCleanup/cache.properties
--- a/.gradle/buildOutputCleanup/cache.properties	(revision b7f554eebc040791c00214c383a21d6a0ae839ca)
+++ b/.gradle/buildOutputCleanup/cache.properties	(date 1632590862246)
@@ -1,2 +1,2 @@
-#Sat Sep 25 20:14:43 EEST 2021
+#Sat Sep 25 20:27:42 EEST 2021
 gradle.version=7.0.2
